src/core/logging/telegramLogger.ts

```diff
@@ -0,0 +1,38 @@
+import axios from "axios";
+
+/**
+ *
+ */
+export class TelegramLogger {
+	private token: String;
+	private chatId: String;
+
+	/**
+	 *
+	 */
+	constructor(token: string, chatId: string) {
+		this.token = token;
+		this.chatId = chatId
+	}
+
+	/**
+	 * Sends the `message` to the telegram chat if token and chatid supplied.
+	 * @param message The message to send.
+	 * @returns boolean if successful
+	 */
+	async sendMessage(message: string) {
+		if (this.token && this.chatId) {
+			try {
+				const { status } = await axios.get(`https://api.telegram.org/bot${this.token}/sendMessage`, { params: { chat_id: this.chatId, text: message } })
```

Comment:
Thoughts on whether to use a library like https://github.com/telegraf/telegraf or https://github.com/yagop/node-telegram-bot-api ? Might provide more connection robustness, error handling and functionality for future use

Reply:
I personally don't like to add a library where 90% of the code is not used. I see the point with the functionality for the future but at the moment we only send messages. If someone wants to add bot functionalities later he can freely choose which library to use.